Project/Src/Gui/TextView.cs

```diff
@@ -69,6 +69,7 @@ public int FirstVisibleLine
         public void Dispose()
         {
             measureCache.Clear();
+            GC.SuppressFinalize(this);
```

Comment:
📝 This call is not necessary. Neither this type nor any derived type is allowed to have a user-defined finalizer because the type does not implement the legacy IDisposable pattern.

Reply:
I was directed by an analyser to add it 😏 
I can't repro on my personal laptop so can't tell which analyser it was.

Reply:
In this case, the analyzer was incorrect. I would go ahead and remove this line and we can identify the problematic analyzer to address it directly.